libs/chain-signatures/contract/tests/ckd.rs

```diff
@@ -210,22 +210,30 @@ async fn test_contract_ckd_fail_refund() -> anyhow::Result<()> {
         .to_string()
         .contains(&errors::RequestError::Timeout.to_string()));
 
-    let new_balance = alice.view_account().await?.balance;
-    let new_contract_balance = contract.view_account().await?.balance;
+    let alice_balance_post_transaction = alice.view_account().await?.balance;
+    let contract_balance_post_transaction = contract.view_account().await?.balance;
     println!(
-        "{} {} {} {}",
-        balance.as_millinear(),
-        new_balance.as_millinear(),
-        contract_balance.as_yoctonear(),
-        new_contract_balance.as_yoctonear(),
+        "alice_balance_pre_transaction: {}\nalice_balance_post_transaction: {}\ncontract_balance_pre_transaction: {}\ncontract_balance_post_transaction: {}",
+        alice_balance_pre_transaction.as_yoctonear(),
+        alice_balance_post_transaction.as_yoctonear(),
+        contract_balance_pre_transaction.as_yoctonear(),
+        contract_balance_post_transaction.as_yoctonear(),
     );
+
     assert!(
-        balance.as_millinear() - new_balance.as_millinear() < 10,
+        alice_balance_pre_transaction.as_millinear()
+            - alice_balance_post_transaction.as_millinear()
+            < 10,
```

Comment:
I know you’re not the author of this magic number, but do you know why we use `10` here?